_posts/2023-09-10-python-multiple-arithmetic-operators.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "The surprising behavior of multiple + and - operators in Python"
+header:
+  image: /assets/images/headers/2023-09-28_pythons-weird-arithmetic.png
+  image_description: "An example of Python's weird arithmetic"
+  teaser: /assets/images/teasers/2023-09-28_pythons-weird-arithmetic.png
+excerpt: Let's explore this strange quirk of the Python language and learn from it.
+---
+
+All programming languages have some quirks and strange behavior.
+Python is certainly no exception.
+I was however quite surprised recently when I found out that Python allows multiple `+` and `-` in a row.
```

Comment:
Cut out the "however". It's a bad habit of mine too. It causes me to lose focus in written text, I think spoken can add emphasis. 

"I was surprised to learn Python"...